tests/requirements-linting.txt

```diff
@@ -4,7 +4,7 @@ flake8-quotes==3.3.1
 hypothesis==6.54.4
 isort==5.10.1
 pyupgrade==2.37.3
-mypy==0.971
+mypy==1.2.0
```

Comment:
Is mypy actually updated to support this? I'm also concerned about errors for older mypy users

Reply:
I'm not a mypy user (IMO it too slow). I use pyright instead. pyright updated their typeshed shortly after the change was made. Looking at the release notes for mypy, it appears they have updated typeshed as well. But yes, it will result it type check failure for users of older versions of mypy the same as it results in failures for those trying to use newer versions of type checkers.